src/components/Navbar-icons/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cn from 'classnames';
+import PropTypes from 'prop-types';
+
+
+export const Icon = ({ iconURL, selected }) => {
+  return (
+    <div className={cn({ 'bg-c800': selected }, 'p-4 w-16 h-16 rounded-lg ')}>
```

Comment:
the background of the element **isn't `c800`**, it's just our blue color but with less opacity
you can add this color as an option at the `tailwind-config.js` file like this
```js
      c100_op: {
        10: 'rgba(112, 72, 232, 0.1)'
      }
```
and then use it instead of the c800